pandas/tests/dtypes/test_common.py

```diff
@@ -532,16 +532,16 @@ def test_is_complex_dtype():
     (float, np.dtype(float)),
     ('float64', np.dtype('float64')),
     (np.dtype('float64'), np.dtype('float64')),
-    pytest.mark.xfail((str, np.dtype('<U')), ),
+    (str, np.dtype(str)),
```

Comment:
On both my system (Windows) and at try.jupyter.org (sys.platform is 'linux', ``_get_dtype(str)`` returns ``np.dtype('<U')``. In the jenkins/appvoyeur,  it returns ``np.dtype('S')``. 

To me it looks like a difference of how numpy is handled on different system and not a pandas issue. I therefore relax that test, to make it only check that ``_get_dtype(str) == np.dtype(str)``, which passes both locally and in the CI system.